submarine-test/e2e/pom.xml

```diff
@@ -108,23 +96,79 @@
             </configuration>
           </execution>
         </executions>
+        <configuration>
+          <argLine>-Xmx2048m</argLine>
+        </configuration>
       </plugin>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <version>2.4</version>
-        <configuration>
-          <archive>
-            <manifest>
-              <mainClass>HelloWorldIT</mainClass>
-            </manifest>
-          </archive>
-          <descriptorRefs>
-            <descriptorRef>jar-with-dependencies</descriptorRef>
-          </descriptorRefs>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration combine.children="append">
+          <argLine>-Xmx2g -Xms1g -Dfile.encoding=UTF-8</argLine>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>start-submarine-server</id>
```

Comment:
When to enter this parameter `start-submarine-server` or  `stop-submarine-server` ?